src/auto-evo/simulation/food_source/FloatingChunkFoodSource.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Linq;
+
+public class FloatingChunkFoodSource : FoodSource
```

Comment:
This name is not accurate because iron are chunks, so this is more accurately just the marine snow thing.